### PR TITLE
Another round of pattern fixes

### PR DESCRIPTION
**Pull request by richtabor**
**Description**

Fixes patterns to make them work better across variations. I uncovered these while working on #574. There should be very minimal visual changes. 

**Screenshots**
CleanShot 2023-10-06 at 14 22 04
CleanShot 2023-10-06 at 14 24 56
CleanShot 2023-10-06 at 14 34 17

### CODE REVIEW

File: patterns/page-04-newsletter-landing.php [hunk 1]
```diff
@@ -29,7 +30,8 @@
 
 <!-- wp:buttons {"layout":{"type":"flex","justifyContent":"center"}} -->
 <div class="wp-block-buttons"><!-- wp:button -->
-<div class="wp-block-button"><a class="wp-block-button__link wp-element-button"><?php echo esc_html_x( 'Sign up', 'sample content for newsletter subscription button', 'twentytwentyfour' ); ?></a></div>
+<div class="wp-block-button"><a class="wp-block-button__link wp-element-button">Sign up</a></div>
```
Comment (carolinan) [inline patterns/page-04-newsletter-landing.php]: Nit: Please make sure that all text is translation ready.

File: patterns/faq.php [hunk 2]
```diff
@@ -2,19 +2,19 @@
 /**
  * Title: Faq
  * Slug: twentytwentyfour/faq
- * Categories: text
+ * Categories: text, about, featured
```
Comment (carolinan) [inline patterns/faq.php:5]: I think this could be easier to find if some keywords were added.

Comment (luminuu) [reply patterns/faq.php:5]: I added some keywords here, but we can still iterate over this in a new PR.

**Review by luminuu — APPROVED** [review]
I added some small fixes and enhancements, LGTM.

File: patterns/page-04-newsletter-landing.php (hunk 1 above)
Comment (luminuu) [reply patterns/page-04-newsletter-landing.php]: I updated this.

**Comment by richtabor** [comment]
Thanks all!

**Pull request merged**